vllm_gaudi/attention/backends/hpu_attn.py

```diff
@@ -205,7 +205,9 @@ def __init__(
             else FP8Matmul()
         self.latent_cache_k = VLLMKVCache() if not self.enable_fp8_attn \
             else VLLMFP8KVCache()
-        self.fused_scaled_dot_product_attention = kernels.fsdpa()
+        HPUFusedSDPA = kernels.fsdpa()
+        self.fused_scaled_dot_product_attention = None if HPUFusedSDPA is None \
+            else ModuleFusedSDPA(HPUFusedSDPA)
```

Comment:
Hmm, that is interesting, I don't recall why we didn't do that before.
Any perf gain you observed with `ModuleFusedSDPA(HPUFusedSDPA)`